doc/index.rst

```diff
@@ -13,7 +13,6 @@ Installation
 
 We recommend the `Anaconda Python distribution <https://www.continuum.io/downloads>`_. To install ``mne_bids``, you first need to install its dependencies::
```

Comment:
The duplicate colon is there so that one is visible when the .rst is rendered??
Also, I added a slightly modified version of your text to the `index.rst` (removed `pandas` since that's kinda the point of this PR...)

Reply:
ohhh okay, thanks for clarifying!